Pod/Classes/TMPodspec.m

```diff
@@ -61,21 +76,125 @@ - (NSString *)version {
     return [self stringForKey:PodspecKeyVersion];
 }
 
-- (NSString *)summary {
-    return [self stringForKey:PodspecKeySummary];
+- (NSArray *)authors {
+    NSString *authorName = [self stringForKey:PodspecKeyAuthor];
+    NSArray *authorNames = [self arrayForKey:PodspecKeyAuthors];
+    NSDictionary *authorNamesAndAddresses = [self dictionaryForKey:PodspecKeyAuthors];
+    
+    NSMutableArray *authors = [NSMutableArray array];
+    
+    if (authorNamesAndAddresses) {
+        [authorNamesAndAddresses enumerateKeysAndObjectsUsingBlock:^(id name, id emailAddress, BOOL *stop) {
+            if ([name isKindOfClass:[NSString class]] && [emailAddress isKindOfClass:[NSString class]]) {
+                TMAuthor *author = [[TMAuthor alloc] initWithName:(NSString *)name emailAddress:(NSString *)emailAddress];
+                [authors addObject:author];
+            }
+        }];
+    }
+    else if (authorNames) {
+        for (id name in authorNames) {
+            if ([name isKindOfClass:[NSString class]]) {
+                TMAuthor *author = [[TMAuthor alloc] initWithName:(NSString *)name emailAddress:nil];
+                [authors addObject:(NSString *)author];
+            }
+        }
+    }
+    else if (authorName) {
+        TMAuthor *author = [[TMAuthor alloc] initWithName:authorName emailAddress:nil];
+        [authors addObject:(NSString *)author];
+    }
+    else {
+        return nil;
+    }
+
+    return authors;
+}
+
+- (NSURL *)socialMediaURL {
+    return [NSURL URLWithString:[self stringForKey:PodspecKeySocialMediaURL]];
+}
+
+- (NSURL *)docsetURL {
+    return [NSURL URLWithString:[self stringForKey:PodspecKeyDocsetURL]];
 }
 
 - (NSURL *)homepageURL {
     return [NSURL URLWithString:[self stringForKey:PodspecKeyHomepageURL]];
 }
 
+- (NSString *)summary {
+    return [self stringForKey:PodspecKeySummary];
+}
+
+- (NSString *)descriptionString {
+    return [self stringForKey:PodspecKeyDescription];
+}
+
+- (NSArray *)screenshotURLs {
+    NSString *singleScreenshotURLString = [self stringForKey:PodspecKeyScreenshot];
+    NSArray *multipleScreenshotURLStrings = [self arrayForKey:PodspecKeyScreenshots];
+
+    NSMutableArray *screenshotURLs = [NSMutableArray array];
+    
+    if (multipleScreenshotURLStrings) {
+        for (id screenshotURLValue in multipleScreenshotURLStrings) {
+            if ([screenshotURLValue isKindOfClass:[NSString class]]) {
+                NSURL *URL = [NSURL URLWithString:(NSString *)screenshotURLValue];
+                if (URL) {
+                    [screenshotURLs addObject:URL];
+                }
+            }
+        }
+    }
+    else if (singleScreenshotURLString) {
+        NSURL *URL = [NSURL URLWithString:singleScreenshotURLString];
+        if (URL) {
+            [screenshotURLs addObject:URL];
+        }
+    }
+    
+    return [screenshotURLs count] ? screenshotURLs : nil;
```

Comment:
I want to represent what the author put in their podsec:

1. If you get back `nil` it means they didn’t include screenshots.
2. If you get back an empty array, it means they tried to but either included them incorrectly or we parsed incorrectly (likely due to some change in the spec).

At least that’s how I thought about it.